src/components/seeds/database/index.tsx

```diff
@@ -368,7 +368,26 @@ export default function Database(props: DatabaseProps): JSX.Element {
         });
 
         if (requestId === getRequestId('accessions_search')) {
-          setSearchResults(apiResponse);
+          const processedResults = apiResponse?.map((result: SearchResponseElementWithId) => {
+            // The "geolocations" value, if present, is a list of child objects each of which has
+            // a "coordinates" field which is a string with comma-separated latitude and longitude.
+            // For display in the table view, we need to turn it into a single string.
+            let coordinatesList = '';
+            (result.geolocations as any[])?.forEach((gl, index) => {
+              if (index === 0) {
+                coordinatesList = gl.coordinates;
+              } else {
+                coordinatesList += `\r${gl.coordinates}`;
+              }
+            });
+
+            return {
+              ...result,
+              'geolocations.coordinates': coordinatesList,
+            };
```

Comment:
This is modeled after the way we handle the list of sub-locations in the nursery inventory table (InventoryListByBatch.tsx).

Reply:
You can handle the table view display by modifying this table's "table cell renderer" [here](https://github.com/terraware/terraware-web/blob/main/src/components/seeds/database/TableCellRenderer.tsx).

Something like this should get you close:
```
if (column.key === 'geolocations.coordinates') {
    const coordinatesValue = row.geolocations?.map((gl) => gl.coordinates).join('\r');
    return <CellRenderer index={index} column={column} value={coordinatesValue} row={row} />;
}
```

Reply:
I suspect the `'\r'` should get replaced with a `', '`, the `\r` won't show up as intended in the UI. Alternatively you could use `<br>` if you are truly trying to break up the text within the table cell.

Reply:
Thanks! Didn't know there was a renderer like that. Should we change the nursery inventory table to use that approach too?

I changed the separator to a semicolon since there are commas in the values. Tried `<br>` but it got escaped, and anyway I don't think we want to make the rows variable-height.

Reply:
Yes, we should change that. I'll make a tech debt ticket, thanks.